java/compaction-status-store/src/test/java/sleeper/compaction/status/job/DynamoDBCompactionJobStatusStoreTimeToLiveIT.java

```diff
@@ -42,7 +44,9 @@ public void shouldSetExpiryDateForCompactionJobStatusWithDefaultTimeToLive() {
         store.jobCreated(job);
```

Comment:
I noticed the store.jobCreated(job) is a pre-condition for the test. It should be under Given rather than When.

The createdTime declaration is a little misleading. The first part is recording the time during a pre-condition, but when you add the time to live that's the expected expiry time, which is a post-condition for the test. Maybe move the .plus(...) to the assertion?

The When is actually store.getJob(job.getId()). Since it makes sense to do that in-line with an assertion it could be under When / Then.